### PR TITLE
Editor decide publishing policy

### PR DESCRIPTION
**Pull request by garshol**
Created publishing_rights.feature
Have adjusted readme and owner_navigate_to_the_landing_page_for_theire_registration (sic!)  accordingly

### CODE REVIEW

File: 1226-editor/publishings_rights.feature [hunk 1]
```diff
@@ -0,0 +1,32 @@
+Feature: Editor decide who got publishing rights
```
Comment (brinxmat) [inline 1226-editor/publishings_rights.feature:1]: decides who gets

File: 1226-editor/publishings_rights.feature [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+	And the Users Feide profile contains one of the following roles:
+		| Scientific employees               |
+		| Technical administrative employees |
+	Then the User is a Registrator
+
+	@needJiraTag 
+	Scenario: Default publishing rights
+	When the Editor of an Institution hasn’t chosen a policy
+	Then the publications policy is:
+		| Registrator got full publishing rights |
```
Comment (brinxmat) [inline 1226-editor/publishings_rights.feature:22]: got -> has?

File: 1226-editor/publishings_rights.feature [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+
+	@needJiraTag 
+	Scenario: Editor decide publishing rights
+	Given a Editor views the Editor page
+	When the Editor chooses one of the three options:
+		| Registrator got full publishing rights |
```
Comment (brinxmat) [inline 1226-editor/publishings_rights.feature:28]: got -> has?

File: 1226-editor/publishings_rights.feature [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+	Then the User is a Registrator
+
+	@needJiraTag 
+	Scenario: Default publishing rights
+	When the Editor of an Institution hasn’t chosen a policy
+	Then the publications policy is:
+		| Registrator got full publishing rights |
+
+	@needJiraTag 
+	Scenario: Editor decide publishing rights
```
Comment (brinxmat) [inline 1226-editor/publishings_rights.feature:25]: decide -> defines

File: 1226-editor/readme [hunk 5]
```diff
@@ -12,9 +12,11 @@ Smaller Institutions may deem one User to be both the Editor and the Curator. A
...
 Some Institutions wish to arrange resources into classes or categories according to shared qualities or characteristics. To accommodate this goal, the Editor may decide which vocabularies the Users may use in the Registration Wizard. Vocabularies are field-/domain-specific, so Institutions with a strong field focus may wish all Users to use the same vocabulary. Others may offer the User a selection of optional vocabularies.
+See editor_opens_editor_administration.feature for details.
 
 Publishing permissions
-Any logged-in User can publish a resource, unless the Editor has set permissions to either: 
-Restrict all publishing, demanding a Curator to review even publication of metadata
-Or Users may publish metadata, but impose a temporary embargo on any files until a Curator has reviewed their content and their licenses.
-The Editor can adjust the restrictions to only cover registration types that are of interest for NVI reporting.
\ No newline at end of file
+The Editor decide the Institutions publication policy and can choose one of the three options:
```
Comment (brinxmat) [inline 1226-editor/readme:18]: decide -> defines

File: 1226-editor/readme [hunk 6]
```diff
@@ -12,9 +12,11 @@ Smaller Institutions may deem one User to be both the Editor and the Curator. A
...
+	Registrator got full publishing rights
```
Comment (brinxmat) [inline 1226-editor/readme:19]: got -> has

File: 1226-editor/readme [hunk 7]
```diff
@@ -12,9 +12,11 @@ Smaller Institutions may deem one User to be both the Editor and the Curator. A
...
+	Registrator can only publish metadata
+	Only Curator can publish
+See publishings_rights.feature for details.
```
Comment (brinxmat) [inline 1226-editor/readme:22]: publishing_rights.feature

File: 1228-landing_page/owner_navigates_to_the_landing_page_for_their_registration.feature [hunk 8]
```diff
@@ -25,42 +25,29 @@ Feature: Owner navigates to the Landing Page for their Resource
         And the user is informed that progress can be viwed in My Messages 
 
     @needJiraTag
-    Scenario Outline: Owner wants to publish Resource, all restrictions
-        Given Owner wants to publish their Resource
-        And Resource is of type "<ResourceType>" 
-        And Editor has restricted publication of type "<ResourceType>" 
+    Scenario: Owner wants to publish Resource, all restrictions
+        Given Institutions publications policy is "Only Curator can publish"
```
Comment (brinxmat) [inline 1228-landing_page/owner_navigates_to_the_landing_page_for_their_registration.feature:29]: publications -> publication

File: 1228-landing_page/owner_navigates_to_the_landing_page_for_their_registration.feature [hunk 9]
```diff
@@ -25,42 +25,29 @@ Feature: Owner navigates to the Landing Page for their Resource
...
         When the Owner uses the Publish option
-        Then an Approval Request is created
-        And the User is informed that a Publishing Approval is pending and progress can be viwed in My Messages
-        Examples:
-            | ResourceType |
-            | NVI          |
-            | none-NVI     |
+        Then the Owner see a landingpage with an unpublished resource
```
Comment (brinxmat) [inline 1228-landing_page/owner_navigates_to_the_landing_page_for_their_registration.feature:31]: the Owner sees the Landing page for the unpublished Resource?

File: 1228-landing_page/owner_navigates_to_the_landing_page_for_their_registration.feature [hunk 10]
```diff
@@ -25,42 +25,29 @@ Feature: Owner navigates to the Landing Page for their Resource
...
+        And an Approval Request is sent to his Curator
```
Comment (brinxmat) [inline 1228-landing_page/owner_navigates_to_the_landing_page_for_their_registration.feature:32]: the user's institutional Curator?

File: 1228-landing_page/owner_navigates_to_the_landing_page_for_their_registration.feature [hunk 11]
```diff
@@ -25,42 +25,29 @@ Feature: Owner navigates to the Landing Page for their Resource
...
+        And the Owner is notified that a Approval Request is sent to his Curator and progress can be viwed in My Messages
```
Comment (brinxmat) [inline 1228-landing_page/owner_navigates_to_the_landing_page_for_their_registration.feature:33]: an Approval…to the Curator

File: 1228-landing_page/owner_navigates_to_the_landing_page_for_their_registration.feature [hunk 12]
```diff
@@ -25,42 +25,29 @@ Feature: Owner navigates to the Landing Page for their Resource
...
 
     @needJiraTag
-    Scenario Outline: Owner wants to publish Resource, file restrictions
-        Given Owner wants to publish their Resource
-        And Resource is of type "<ResourceType>" 
-        And the Editor has restricted access to files of Resources of "<ResourceType>"
+    Scenario: Owner wants to publish Resource, file restrictions
+        Given Institutions publications policy is "Registrator can only publish metadata"
```
Comment (brinxmat) [inline 1228-landing_page/owner_navigates_to_the_landing_page_for_their_registration.feature:37]: Institution(al)? publication, in all cases

Comment (brinxmat) [reply 1228-landing_page/owner_navigates_to_the_landing_page_for_their_registration.feature:37]: I don't think you need "is"

File: 1228-landing_page/owner_navigates_to_the_landing_page_for_their_registration.feature [hunk 13]
```diff
@@ -25,42 +25,29 @@ Feature: Owner navigates to the Landing Page for their Resource
...
         When the Owner uses the Publish option
-        Then an extra embargo is put on all files
+        Then the Owner sees a landingpage with a Published Resource
```
Comment (brinxmat) [inline 1228-landing_page/owner_navigates_to_the_landing_page_for_their_registration.feature:39]: See the previous comment about this

File: 1228-landing_page/owner_navigates_to_the_landing_page_for_their_registration.feature [hunk 14]
```diff
@@ -25,42 +25,29 @@ Feature: Owner navigates to the Landing Page for their Resource
...
-        And the Landing Page is publicly accessible
-        And an Approval Request is created
-        And the User is informed that the Resource is published but a Publishing Approval is pending to allow access to the files and progress can be viwed in My Messages
-        Examples:
-        Examples:
-            | ResourceType |
-            | NVI          |
-            | none-NVI     |
+        And the Resource's content is locked with a pending approval notification
+        And an Approval Request is sent to his Curator
```
Comment (brinxmat) [inline 1228-landing_page/owner_navigates_to_the_landing_page_for_their_registration.feature:42]: the Curator

File: 1228-landing_page/owner_navigates_to_the_landing_page_for_their_registration.feature [hunk 15]
```diff
@@ -25,42 +25,29 @@ Feature: Owner navigates to the Landing Page for their Resource
...
+        And the Owner is notified that a Approval Request is sent to his Curator and progress can be viwed in My Messages
```
Comment (brinxmat) [inline 1228-landing_page/owner_navigates_to_the_landing_page_for_their_registration.feature:43]: See previous comment about this line

File: 1228-landing_page/owner_navigates_to_the_landing_page_for_their_registration.feature [hunk 16]
```diff
@@ -25,42 +25,29 @@ Feature: Owner navigates to the Landing Page for their Resource
...
 
     @needJiraTag
     Scenario: Owner uses the Publish option on Langing Page
-        Given Owner wants to publish their Resource
-        And the Editor has given all users publishing permissions for their own work
+        Given Institutions publications policy is "Registrator got full publishing rights"
```
Comment (brinxmat) [inline 1228-landing_page/owner_navigates_to_the_landing_page_for_their_registration.feature:47]: Again

File: 1228-landing_page/owner_navigates_to_the_landing_page_for_their_registration.feature [hunk 17]
```diff
@@ -25,42 +25,29 @@ Feature: Owner navigates to the Landing Page for their Resource
...
         When the Owner uses the Publish option
         Then the Resource's status is "Published"
-        And the Landing Page is publicly accessible
+        And the Owner see a landingpage with a published resource
```
Comment (brinxmat) [inline 1228-landing_page/owner_navigates_to_the_landing_page_for_their_registration.feature:50]: the Owner sees a landing page with a published Resource.

File: 1228-landing_page/owner_navigates_to_the_landing_page_for_their_registration.feature [hunk 18]
```diff
@@ -25,42 +25,29 @@ Feature: Owner navigates to the Landing Page for their Resource
         And the user is informed that progress can be viwed in My Messages 
 
     @needJiraTag
-    Scenario Outline: Owner wants to publish Resource, all restrictions
-        Given Owner wants to publish their Resource
-        And Resource is of type "<ResourceType>" 
-        And Editor has restricted publication of type "<ResourceType>" 
+    Scenario: Owner wants to publish Resource, all restrictions
```
Comment (brinxmat) [inline 1228-landing_page/owner_navigates_to_the_landing_page_for_their_registration.feature:28]: In this file, I feel we need to have consistent capitalisation of NVA concepts. Resource, maybe Published Resource and Landing page

**Review by brinxmat — CHANGES_REQUESTED** [review]

File: 1226-editor/readme (hunk 6 above)
Comment (brinxmat) [reply 1226-editor/readme:19]: Still… :D

**Review by brinxmat — CHANGES_REQUESTED** [review]

**Review by brinxmat — APPROVED** [review]

**Review by terjehell — APPROVED** [review]

**Pull request merged**